src-tauri/Cargo.toml

```diff
@@ -34,7 +34,7 @@ gifski = { version = "1.12.2", default-features = false, features = [
 	"png",
 	"gifsicle",
 ] }
-helmer-media = { git = "ssh://git@github.com/helmerapp/helmer-media.git" }
+helmer-media = { git = "https://github.com/helmerapp/helmer-media.git", branch = "master" }
```

Comment:
Let's keep it ssh. I'm not sure if https would work correctly for all, since `helmer-media` is a private repo